data/pokemon_upgrade_cost.json

```diff
@@ -0,0 +1,82 @@
+[
+[1, 200],
+[1, 200],
```

Comment:
According to my data, the price change every 4 pokemon level. I am not sure data is correct though, need to double check.